src/server/pfs/server/driver.go

```diff
@@ -130,10 +131,22 @@ func newDriver(
 		return nil, errors.Errorf("cannot initialize driver with nil treeCache")
 	}
 	// Initialize driver
+	objClient, err := obj.NewClientFromSecret(storageRoot)
+	if err != nil {
+		return nil, err
+	}
+	if env.StorageDiskCacheSize > 0 {
+		diskCache, err := obj.NewLocalClient(filepath.Join(os.TempDir(), "pfs-cache", uuid.NewWithoutDashes()))
```

Comment:
@Tryneus Does the tmp directory seem like a reasonable place for this as compared to `StorageRoot` or `CacheRoot`?

Reply:
This seems fine to me